src/web_app_skeleton/bin/setup

```diff
@@ -3,6 +3,18 @@
 # Exit if any subcommand fails
 set -e
 
+if ! command -v yarn > /dev/null; then
+  printf 'Yarn is not installed.\n'
+  printf 'See https://yarnpkg.com/lang/en/docs/install/ for install instructions.\n'
+  exit 1
+fi
+
+if ! command -v brunch > /dev/null; then
+  printf 'Brunch is not installed.\n'
+  printf 'Run "npm install -g brunch" to install brunch.\n'
+  exit 1
+fi
```

Comment:
You don't need this because if you have yarn then `yarn install` will install it. `bin/compile_assets` uses the brunch version installed in the project directory, so you don't need to install it globally

Reply:
👍